### PR TITLE
BAEL-4539: Fix failing tests for Vault

### PR DESCRIPTION
**Pull request by tapankavasthi**
## Summary
- Fix tests for Spring Vault Module

## Jira
[BAEL-4539](https://jira.baeldung.com/browse/BAEL-4539)

## Test Results
Screenshot 2023-03-13 at 8 14 25 PM

### CODE REVIEW

File: spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationManualTest.java [hunk 1]
```diff
@@ -1,90 +1,76 @@
...
+        // When
+        credentialsService.secureCredentials(actualCredentials);
         // Then
-        assertNotNull(credentials);
-        assertEquals("username", credentials.getUsername());
-        assertEquals("password", credentials.getPassword());
+        Credentials storedCredentials = credentialsService.accessCredentials(actualCredentials.getUsername());
+        Assertions.assertNotNull(storedCredentials);
+        Assertions.assertEquals(actualCredentials.getUsername(), storedCredentials.getUsername());
+        Assertions.assertEquals(actualCredentials.getPassword(), storedCredentials.getPassword());
```
Comment (KevinGilmore) [inline spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationManualTest.java:64]: The order of arguments should be swapped for these assertEquals calls -- they should be assertEquals(expected, actual).

File: spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationManualTest.java [hunk 2]
```diff
@@ -96,23 +82,22 @@ public void givenCredentials_whenSave_thenReturnCredentials() {
...
-        assertNotNull(returnedCredentials);
-        assertNotNull(returnedCredentials.get());
-        assertEquals(returnedCredentials.get()
-            .getUsername(), credentials.getUsername());
-        assertEquals(returnedCredentials.get()
-            .getPassword(), credentials.getPassword());
+        assertNotNull(retrievedCredentials);
+        assertNotNull(retrievedCredentials.get());
+        assertEquals(retrievedCredentials.get()
+            .getUsername(), expectedCredentials.getUsername());
```
Comment (KevinGilmore) [inline spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationManualTest.java:98]: Swap order of arguments.

**Review by KevinGilmore — COMMENTED** [review]
A few assertEquals calls have the order of arguments reversed.

File: spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationManualTest.java (hunk 1 above)
Comment (tapankavasthi) [reply spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationManualTest.java]: Thanks, @KevinGilmore, for catching this. I've fixed the issue.

File: spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationManualTest.java (hunk 2 above)
Comment (tapankavasthi) [reply spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationManualTest.java]: Thanks, @KevinGilmore, for catching this. I've fixed the issue.

**Pull request merged**